ios/Exponent/Versioned/Core/Internal/EXAppState.m

```diff
@@ -80,10 +80,10 @@ - (void)setState:(NSString *)state
       [self sendEventWithName:@"appStateDidChange"
                          body:@{@"app_state": _lastKnownState}];
     }
+   
     // change state on universal modules
     // TODO: just make EXAppState a universal module implementing UMAppLifecycleService
-    id<UMAppLifecycleService> lifeCycleManager = [self.bridge.scopedModules.moduleRegistry
-                                                  getModuleImplementingProtocol:@protocol(UMAppLifecycleService)];
+    id<UMAppLifecycleService> lifeCycleManager = [[[self.bridge moduleForClass:[UMModuleRegistryHolderReactModule class]] moduleRegistry] getModuleImplementingProtocol:@protocol(UMAppLifecycleService)];
```

Comment:
It'll be hard to do because we need to separate also this  https://github.com/expo/expo/commit/03313113bce04ced5c39ded21698da5b2cf8bb83. 
Unfortunately, this commit is related to this PR and contains a lot of code.